tools/composer.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Does renovate need adjustments here too? 🤔

Reply:
I was hoping it will work automatically 🤞